lib/loading_screen.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:distal_radius/image_upload_screen.dart';
+import 'package:flutter/material.dart';
+
+import 'package:camera/camera.dart';
+import "camera_screen.dart";
+import 'screen_button.dart';
+import "menu_screen.dart";
+
+class LoadingScreen extends StatefulWidget {
+  const LoadingScreen({Key? key}) : super(key: key);
+
+  //String to be used in navigator
+  static const String id = "loading_screen";
+
+  @override
+  State<LoadingScreen> createState() => _LoadingScreenState();
+}
+
+class _LoadingScreenState extends State<LoadingScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Container(
+          margin: const EdgeInsets.fromLTRB(0, 35, 0, 0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: <Widget>[
+              Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                //back button
+                  Stack(alignment: Alignment.center, children: const [
+                    Positioned(left: 10, child: BackButton()),
```

Comment:
I would remove this back arrow button, and instead, have a "Cancel" button at the bottom of the screen to make the action more clear.

Reply:
I think the button is fine, we can leave it as is for the time being